CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 # 0.19.0
 - added `statsd_prefix` option to `netuitive_configure`
 
+- added `batch_size` @johvet
```

Comment:
You didn't merge this correctly. This line should have gone under `[Unreleased]` but it ended up under 0.19.0